src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -175,7 +175,7 @@ public DocumentDatabase(string name, RavenConfiguration configuration, ServerSto
                 {
                     serverStore.DatabasesLandlord.CatastrophicFailureHandler.Execute(name, e, environmentId, environmentPath, stacktrace);
                 });
-                _hasClusterTransaction = new AsyncManualResetEvent(DatabaseShutdown);
+                _hasClusterTransaction = new ManualResetEvent(false);
```

Comment:
`ManualResetEventSlim` most of the time is a better choice

Reply:
done